### ISSUE TITLE
PRACK send with an undefined rseq value in RAck header

### ISSUE THREAD

**Issue by FThrum** [onsip/SIP.js#672]
In some scenarios I see that SIP.js sends a PRACK, but the RAck header  contains "undefined" as rseq number.
The an resq does not exist in in the responses before.

Those are the relevant sip headers for an outgoing call from SIP.js 0.13.5, I removed the headers which are not relevant:
```
->
INVITE sip:123456789@null SIP/2.0
CSeq: 2641 INVITE
Allow: ACK,CANCEL,INVITE,MESSAGE,BYE,OPTIONS,INFO,NOTIFY,REFER
Supported: outbound
Contact: <sip:123@g2rohmtp9vaq.invalid;transport=wss;ob>

<-
SIP/2.0 183 Session Progress
CSeq: 2641 INVITE
Supported: replaces
Allow: INVITE, ACK, CANCEL, OPTIONS, BYE, REFER, SUBSCRIBE, NOTIFY, INFO, PUBLISH, MESSAGE
Contact: <sip:456@10.111.222.1:5060;transport=udp>

(with SDP)

<-
SIP/2.0 180 Ringing
CSeq: 2641 INVITE
Supported: replaces
Allow: INVITE, ACK, CANCEL, OPTIONS, BYE, REFER, SUBSCRIBE, NOTIFY, INFO, PUBLISH, MESSAGE
Contact: <sip:456@10.111.222.1:5060;transport=udp>

->
PRACK sip:456@10.111.222.1:5060;transport=udp SIP/2.0
CSeq: 2642 PRACK
Supported: outbound
RAck: undefined 2641 INVITE

<-
SIP/2.0 400 Bad Request
Warning: 399 sip "Syntax error in RAck"
```

**To Reproduce (if possible)**
Make a outbound call to a destination which first answers with 183 Session progress and than with 180 Ringing.
Not sure if this is timing related.
The issue does not happen if there is a 180 Ringing and then an 183 SDP (no PRACK is send as all)

**Expected behavior**
The PRACK should not be send at all, because the other side does not Indicate 100rel as Supported  or as Required.

**Observed behavior**
A PRACK is send even it the rseq is unknown

**Environment Information**
Chrome 73 on MAC OS.

**Additional context**
As a quick fix the could should check if the presense of the rseq before sending any PRACK.

In SIP.js 0.13.7:
```
22690             extraHeaders.push("RAck: " + response.getHeader("rseq") + " " + response.getHeader("cseq"));
22691             this.earlyDialogs[id].pracked.push(response.getHeader("rseq"));
22692             this.earlyDialogs[id].sendRequest(this, Constants_1.C.PRACK, {

```
The sendRequest should be only send if  response.getHeader("rseq") returns something.

**Comment by egreenmachine** [comment]
We believe that this should be resolved in `0.14.1`. Can you please test and let us know?

**Comment by james-criscuolo** [comment]
Closing this due to inactivity. Feel free to re-open or open a new issue if the issue persists, but we believe the issue has been resolved since `0.14.0` (suggested version is currently `0.14.2`).
